definitions/variable/technology/electricity-operation.yaml

```
@@ -91,3 +92,19 @@
     description: Marginal Cost associated to the Frequency Containment requirement
     unit: EUR_2020/MWh
     skip-region-aggregation: true
+    
+- Export|Electricity:
+    description: Exported electricity from a given region
+    unit: MWh
+- Import|Electricity:
+    description: Exported electricity from a given region
+    unit: MWh
+- Non served Demand|Electricity:
+    description: Not served demand
+    unit: MWh
+- Charged Energy|Electricity|{Electricity Input}:
```

I'm wondering if "Charged Energy" is sufficiently intuitive, and the term also doesn't quite work for other erngy carriers than electricty. Maybe use "Net Injection into Storage" (and use a negative value if more energy is discharged than charged)?

---

For me, "Charged Energy" is intuitive but would be associated with the new storage type tag when it is merged.

---

I also agree with @erikfilias , but Net Injection into Storage is even better , maybe could be simplified into Injected Energy? (then we would have Injected Energy and Discharged Energy?

---

This suggestion implements the name change above, and it changes the tag-list from the energy-input-fuels to the storage types (so that we can track to which storage type energy is injected/discharged).
```suggestion
- Injected Energy|Electricity|{Electricity Storage Type}:
```

---

ok , Injected sounds better :-)